lms/djangoapps/learner_dashboard/views.py

```diff
@@ -77,11 +78,19 @@ def program_details(request, program_uuid):
 
     if waffle.switch_is_active('new_program_progress'):
         course_progress = meter.progress(programs=[program_data], count_only=False)[0]
+
+        sidebar_data = {
+            # TODO: Reuse course_progress?
+            'progress': meter.progress(programs=[program_data])[0]
```

Comment:
Yeah, what's the difference between ```course_progress``` and ```sidebar_data.progress```? It would be nice if we could just have 1 progress object, then ```certificates``` could move up a level.